### PR TITLE
[MOSIP-28022] Enhanced the property migrator - property category csv …

### PR DESCRIPTION
**Pull request by Mohanraj209**
…to include properties from all the modules.

### CODE REVIEW

File: deployment/v3/utils/prop_migrator/knowledge/property-categories.csv [hunk 1]
```diff
@@ -1,4 +1,760 @@
...
+admin-default.properties,mosip.admin.security.policy.userrole-auth-url,URL
+admin-default.properties,mosip.admin.uinmgmt.uin-detail-search,URL
+admin-default.properties,mosip.mosip.kernel.packet-status-update-url,URL
+admin-default.properties,mosip.kernel.packet-reciever-api-url,URL
+admin-default.properties,mosip.kernel.zone-validation-url,URL
+admin-default.properties,mosip.kernel.audit.manager.api,URL
+admin-default.properties,mosip.kernel.masterdata.audit-url,URL
+admin-default.properties,mosip.admin-services.audit.manager.api,URL
+admin-default.properties,mosip.open-id.base-url,URL
+admin-default.properties,mosip.admin-services.open-id,clientsecret,Key
```
Comment (vishwa-vyom) [inline deployment/v3/utils/prop_migrator/knowledge/property-categories.csv:22]: Check this it has comma before and after the clientsecret

File: deployment/v3/utils/prop_migrator/knowledge/property-categories.csv [hunk 2]
```diff
@@ -1,4 +1,760 @@
...
+admin-default.properties,mosip.admin-services.redirecturi,URL
+admin-default.properties,mosip.admin-services.open-id.authorization_endpoint,URL
+admin-default.properties,mosip.admin-services.open-id.token_endpoint,URL
```
Comment (vishwa-vyom) [inline deployment/v3/utils/prop_migrator/knowledge/property-categories.csv:25]: I see the below as duplicate but from different files, can you check this once ?

syncdata.websub.callback.secret.ca-cert

Also generally ensure there are no duplicate in these files

File: deployment/v3/utils/prop_migrator/knowledge/property-categories.csv [hunk 3]
```diff
@@ -1,4 +1,760 @@
...
+id-authentication-default.properties,ida-websub-partner-service-partner-api-key-updated-callback-relative-url,URL
+id-authentication-default.properties,ida-websub-partner-service-misp-license-generated-callback-relative-url,URL
+id-authentication-default.properties,ida-websub-partner-service-misp-license-updated-callback-relative-url,URL
+id-authentication-default.properties,ida-websub-partner-service-oidc-client-created-callback-relative-url,URL
+id-authentication-default.properties,ida-websub-partner-service-oidc-client-updated-callback-relative-url,URL
+id-authentication-default.properties,mosip.kernel.keymanager.hsm.keystore-pass,Key
+id-authentication-default.properties,mosip.iam.adapter.clientsecret,Key
+id-authentication-default.properties,keymanager_database_url,URL
+id-authentication-default.properties,keymanager_database_password,Key
+id-authentication-default.properties,cred-request-service-get-request-ids.rest.uri,Key
```
Comment (vishwa-vyom) [inline deployment/v3/utils/prop_migrator/knowledge/property-categories.csv:223]: Is this really a key ? Looks like a URL

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: deployment/v3/utils/prop_migrator/knowledge/property-categories.csv (hunk 1 above)
Comment (Mohanraj209) [reply deployment/v3/utils/prop_migrator/knowledge/property-categories.csv:22]: updated sir

File: deployment/v3/utils/prop_migrator/knowledge/property-categories.csv (hunk 2 above)
Comment (Mohanraj209) [reply deployment/v3/utils/prop_migrator/knowledge/property-categories.csv:25]: The above property file is present in "kernel-default.properties" and "syncdata-default.properties" sir, i have cross verified.

File: deployment/v3/utils/prop_migrator/knowledge/property-categories.csv (hunk 3 above)
Comment (Mohanraj209) [reply deployment/v3/utils/prop_migrator/knowledge/property-categories.csv:223]: updated sir

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**